foreman/foreman.spec

```diff
@@ -724,6 +738,12 @@ install -d -m0755 %{buildroot}%{_localstatedir}/lib/%{name}/tmp/pids
 install -d -m0755 %{buildroot}%{_localstatedir}/run/%{name}
 install -d -m0750 %{buildroot}%{_localstatedir}/log/%{name}
 install -d -m0750 %{buildroot}%{_localstatedir}/log/%{name}/plugins
+#Copy init scripts and sysconfigs
+install -Dp -m0644 %{buildroot}%{gem_instdir}/%{deploy}/%{tasks_service_name}.sysconfig %{buildroot}%{_sysconfdir}/%{tasks_service_name}
```

Comment:
Correct - that's done on https://github.com/theforeman/foreman/pull/4316/files now